test/test.dart

```diff
@@ -154,7 +159,23 @@ void parseTestNameFromArguments(List<String>? arguments) {
   }
 
   int nameArgIndex = arguments.indexOf("--name");
-  if (nameArgIndex >= 0 && arguments.length > 1) {
+  if (nameArgIndex >= 0 && arguments.length > nameArgIndex) {
```

Comment:
What if `nameArgIndex` == `arguments.length - 1`?

Reply:
That would mean that whoever ran the tests didn't actually pass in the name? The way I read the original code, the arguments would look like `*args* --name someName *more args*`. With the way it was written, we would get a crash if someone ran the command with `*args* --name` - was that desirable?

Reply:
You are right, I believe the args package will bail before we get here